invenio_config/default.py

```diff
@@ -34,3 +34,45 @@ def init_app(self, app):
             warnings.warn(
                 'Set configuration variable SECRET_KEY with random string',
                 UserWarning)
+
+        ALLOWED_HTML_TAGS = app.config.get('ALLOWED_HTML_TAGS')
+
+        if ALLOWED_HTML_TAGS is None:
+            app.config['ALLOWED_HTML_TAGS'] = [
```

Comment:
This seems a bit cluttered, what about extracting it to a variable? you could also use this variable in the tests, instead of rewriting the whole list.